spec/abci++/abci++_app_requirements_002_draft.md

```diff
@@ -123,13 +136,13 @@ Additionally,
 >**TODO** I have left out all the "events" as they don't have any impact in safety or liveness
 >(same for consensus params, and validator set)
 
-* Requirement 11 [`FinalizeBlock`, determinism-1]: For any correct process $p$,
+* Requirement 12 [`FinalizeBlock`, determinism-1]: For any correct process $p$,
   $s_{p,h}$ exclusively depends on $s_{p,h-1}$ and $v_{p,h}$.
 
-* Requirement 12 [`FinalizeBlock`, determinism-2]: For any correct process $p$,
+* Requirement 13 [`FinalizeBlock`, determinism-2]: For any correct process $p$,
   the contents of $T_{p,h}$ exclusively depend on $s_{p,h-1}$ and $v_{p,h}$.
 
-Note that Requirements 11 and 12, combined with Agreement property of consensus ensure
+Note that Requirements 12 and 13, combined with Agreement property of consensus ensure
```

Comment:
I guess this renumbering was a pain. Also for future extensions, perhaps we should give names to the requirements such that adding new ones does not change the names of the old ones.

Reply:
Good idea. Indeed, now that we are merging, I don't think it'll be a good idea to re-number again, so what you propose is probably what will happen if we need to extend the properties after this PR is merged.